deployment/scripts/uuid_migrate.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+
+set -ex
```

Comment:
`set -xeuo pipefail` - for pipefail, see below - `-u` means to error (and exit with `-e`) if you try to read an unset variable.  It is good for catching spelling errors and some logic errors.